test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -0,0 +1,232 @@
+module UnitTest {
+  use Reflection;
+  /*
+    Assert that a boolean condition is true.  If it is false, prints
+    'assert failed' and halts the program.
+
+    :arg test: the boolean condition
+    :type test: `bool`
+  */
+  pragma "insert line file info"
+  pragma "always propagate line file info"
+  proc assertTrue(test: bool) {
+    if !test then
+      __primitive("chpl_error", c"assert failed");
+  }
+
+  /*
+    Assert that a boolean condition is false.  If it is true, prints
+    'assert failed' and halts the program.
+
+    :arg test: the boolean condition
+    :type test: `bool`
+  */
+  pragma "insert line file info"
+  pragma "always propagate line file info"
+  proc assertFalse(test: bool) {
```

Comment:
nitpick: I noticed 3 styles of `:` used throughout code:

```
val: type
val : type
val:type
```

I would suggest staying consistent across the file. Most developers tend to use one of the first 2 styles.

Reply:
I am using the first style and have made necessary changes.